lms/djangoapps/labster_course_license/models.py

```diff
@@ -17,26 +22,46 @@ class CCXLocatorField(OpaqueKeyField):
 
 class CourseLicense(models.Model):
     """
-    A Labster License.
+    A Labster License with related simulations for course.
     """
-    course_id = CCXLocatorField(max_length=255, db_index=True, unique=True)
+    course_id = CCXLocatorField(max_length=255, db_index=True, unique=True, help_text="CCXCourse locators only")
     license_code = models.CharField(max_length=255, db_index=True)
+    simulations = JSONField(help_text="List of course licensed simulations, stored as json string value")
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    modified_at = models.DateTimeField(auto_now=True, null=True)
 
     @classmethod
     def get_license(cls, course_id):
+        """
+        Return license by course key.
+        """
         try:
             return cls.objects.get(course_id=course_id)
-        except cls.DoesNotExist:
+        except ObjectDoesNotExist:
```

Comment:
because for some reason 39 line could return DoesNotExist object and we get `DoesNotExist object is not callable` exception